packages/astro/src/server/clerk-middleware.ts

```diff
@@ -279,10 +277,6 @@ async function decorateRequest(
             controller.enqueue(clerkAstroData);
             controller.enqueue(clerkSafeEnvVariables);
 
-            if (__HOTLOAD__) {
-              controller.enqueue(hotloadScript);
-            }
-
```

Comment:
We are no longer attaching the hotload script to the Response since we're using the `loadClerkJsScript` shared function in client instead

Reply:
You can still use `loadClerkJsScript` in the client but we will expect to read the existing script.